main/channel.c

```diff
@@ -712,12 +718,23 @@ static int does_id_conflict(const char *uniqueid)
 {
 	struct ast_channel *conflict;
 	size_t length = 0;
+	struct ast_channel_callback_data callback_data = {.search = uniqueid, .final_match_result = 0};
```

Comment:
Can we put each element on its own line? Not sure if that's explicitly in the coding guidelines but that's easier to read.

Reply:
Fixed